src/lib.rs

```diff
@@ -37,39 +37,81 @@ mod macos;
 use macos as platform;
 
 use std::env;
-use std::fs::File;
+use std::ffi::OsStr;
+use std::fs::{self, File};
 use std::io::BufReader;
 use std::io::{Error, ErrorKind};
 use std::path::{Path, PathBuf};
 
 use pki_types::CertificateDer;
 
+const ENV_CERT_FILE: &str = "SSL_CERT_FILE";
+const ENV_CERT_DIR: &str = "SSL_CERT_DIR";
+
 /// Load root certificates found in the platform's native certificate store.
 ///
-/// If the SSL_CERT_FILE environment variable is set, certificates (in PEM
-/// format) are read from that file instead.
+/// ## Environment Variables
+///
+/// | Env. Var.      | Description                                                         |
+/// |----------------|---------------------------------------------------------------------|
+/// | SSL_CERT_FILE  | File containing an arbitrary number of certificates in PEM format.  |
+/// | SSL_CERT_DIR   | Directory utilizing the hierarchy used by OpenSSL's [c_rehash].     |
```

Comment:
I might be tempted to say "hierarchy and naming convention used by ..." here to emphasize the file names are a wonky hash based doo-dad

Reply:
Done